bgp/spine.conf

```
@@ -0,0 +1,22 @@
+hostname emilien-bgp-spine-gateway
+frr defaults traditional
+log syslog informational
+!
+interface eth0
+ ip address 192.168.0.3/24
+exit
+!
+router bgp 65000
+ bgp fast-convergence
+ bgp router-id 192.168.0.3
+ no bgp ebgp-requires-policy
+ neighbor 192.168.0.10 remote-as 65001
+ neighbor 192.168.0.20 remote-as 65002
+ neighbor 192.168.0.30 remote-as 65003
```

Because the spine router is multihomed, you need to force it to use the right interface when establishing the peering session:
```suggestion
 neighbor 192.168.0.10 remote-as 65001
 neighbor 192.168.0.10 update-source eth0
 neighbor 192.168.0.20 remote-as 65002
 neighbor 192.168.0.10 update-source eth0
 neighbor 192.168.0.30 remote-as 65003
 neighbor 192.168.0.30 update-source eth0
```